modules/learning-deploying-application-storage-viewing.adoc

```diff
@@ -5,8 +5,11 @@
 [id="learning-deploying-application-storage-viewing_{context}"]
 = Viewing a persistent volume claim
 
+[role="_abstract"]
+You can view your application's persistence volume claim from (cluster-manager-url).
```

Comment:
the attribute did not resolve in the preview:
<img width="827" height="236" alt="image" src="https://github.com/user-attachments/assets/e8044cae-23a4-4137-b4f5-92922624cb76" />